wallet/wallet.go

```diff
@@ -870,6 +870,8 @@ func (w *Wallet) syncWithChain(chainClient *chain.RPCClient) error {
 		}
 	}
 
+	w.resendUnminedTxs()
```

Comment:
since the caller already has the client, just pass that in as a parameter here instead of looking it up again through the wallet inside the function.